airflow/www/views.py

```diff
@@ -2288,6 +2291,36 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, LoggingMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except Exception as e:
+            self.logger.error(e)
+            pass
+
+        # Get the Git repo and git hash
+        git_proj_uri = git_repo = git_hash = None
+        git_version = None
+        try:
+            with open("airflow/git_version") as f:
```

Comment:
Suggestion? Define the path in a common place and import the path? Where's a good place to define it?